### DISCUSSION TITLE
Which features depend on plugin or template?

### DISCUSSION THREAD

**Discussion by andrey-jef** [ObsidianPublisher/obsidian-github-publisher#233] [Q&A — Plugin] [answered]
Hi Mara,

I'm curious to learn the dependency of the features of plugin (for example: convert wikilinks, convert dataview query into static html, etc.)?

It means that these features can be used by installing the plugin in Obsidian only, without the needs of cloning the netlify-template (I can create my own mkdocs local then push to github)? Or I must use both the plugin and cloning your template?

As my current understanding,

These features depends only on the plugin
- convert wikilinks
- convert dataview query

These features are supported by mkdocs-material (I must clone the template, or copy the config from template repo into my own mkdocs repo)
- callout
- latex formula

**Comment by Mara-Li** [comment, marked as answer]
Hello!
It's a good question :)! 

Callout is a plugin for mkdocs! It is not from directly material-mkdocs.

Latex is supported basically by mkdocs (I think?) or/and by Material Mkdocs.

Embed (without using the bake option in the plugin) is my own plugin for mkdocs.

Pure wikilinks is a combo between a plugin for mkdocs and my own Javascript.
It's a little cursed because the original plugin is complicated to edit and I don't want to add more time in the build, so I created the JS instead of edit the plugin...

But, someone work on a plugin for mkdocs that will take everything for Obsidian and we work together to make it compatible with the plugin in Obsidian (notably the relative links I use for the links conversion)

**Reply by andrey-jef** [reply]
Nice to hear the collab with another dev for mkdocs plugin.

So as u wrote, if i don't clone your template but create my own, then some features (like wikilinks conversion) might breaks, but others (like dataview query transform, latex) will be good to go. 

Thanks for your insight

**Reply by Mara-Li** [reply]
Will love to have a share showcase of your future template!

**Reply by andrey-jef** [reply]
Oh. Nothing fancy about my template :)) 

I was just being anxious with multiple daily failed message from the github action, which I clone from your repo to update the template. So i was curious if I stop using/keeping up to date with your template repo, which features might not function as expected. 

As u pointed in your previous answer, the wikilinks conversion might be broken due to the mingled transforming code in both plugin and repo for this feature, while the others (like dataview, latex and callout) will work normally
